### ISSUE TITLE
Blocker to new sign up

### ISSUE THREAD

**Issue by Dr-Bernard** [RMNCLDYO/claude-ai-toolkit#5]
**Describe the bug**

I noticed that after registering on the User interface with gmail account there is another requirement to confirm phone number. However whenever an MTN Nigeria Phone number is added to generate the 

**To Reproduce**
1. Go to the URL: 'https://claude.ai/login?returnTo=%2F%3F'
2. Input a valid google email account as indicated. Then Click on 'continue with email'. This will open a new Claude.ai page prompting you to 'Enter verification code'.
3. Go to your email inbox, open the email from 'support@mail.anthropic.com' and click on the button named 'Sign in to Claude.ai' within the body of the email. This will open a new Claude.ai page containing 'Use verification code to continue'. Copy the verification code on this page and put in the Claude.ai page in step two above then click 'Verify Email Address' button. This will now open a new 'Claude.ai' page with the URL: 'https://claude.ai/onboarding?returnTo=%2F%3F'.
4. Choose 'Nigeria' from the drop down menu of country codes then put a valid Nigerian phone number such as the MTN number: +2348037835105 (or in the format: 08037835105), check the box 'I confirm that I am at least 18 years of age' then click on the button 'Send Verification Code'. A new page will open promoting you to put the verification code sent to the phone then click 'Verify and Create account'
5. See error: an error message is displayed saying 'put a valid phone number'.

**Comment by Dr-Bernard** [comment]
Find here attached the error message I was receiving
![1 Screenshot 2024-06-27 124813](https://github.com/RMNCLDYO/claude-ai-toolkit/assets/57553165/ead0dd8f-3845-4db8-87c1-f58f6a9318b9)
![2 Screenshot 2024-06-27 124837](https://github.com/RMNCLDYO/claude-ai-toolkit/assets/57553165/13a11e00-0496-40f4-a2f5-a48b020f2201)

**Comment by Dr-Bernard** [comment]
Wow.

That was fast.

I just tried it again and it was successful. I am extremely impressed by the speedy resolution of this issue.

As shown in the attached screenshots, I have now opened a new Claude.ai account using the same Nigerian MTN phone number in my mobile phone after several trials since last week.

Thanks guys!
![a Screenshot 2024-06-27 125806](https://github.com/RMNCLDYO/claude-ai-toolkit/assets/57553165/0f9d6eac-d67e-4c10-948e-d8b16fe3d023)
![b Screenshot 2024-06-27 125836](https://github.com/RMNCLDYO/claude-ai-toolkit/assets/57553165/48a2295a-a437-4683-8aca-3e40f992dd46)
